app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -24,3 +26,70 @@ $govuk-cookie-banner-background: #d5e8f3;
     display: inline-block;
   }
 }
+
+// Styles for the new version of the cookie banner
+.gem-c-cookie-banner--new {
+  @include govuk-font($size: 16);
+  border: 2px solid govuk-colour("black");
+}
+
+.gem-c-cookie-banner__wrapper.js-hidden {
+  display: none;
+}
+
+.gem-c-cookie-banner--new .gem-c-cookie-banner__message {
+  display: inline-block;
+  margin-bottom: govuk-spacing(1);
+  padding-right: govuk-spacing(4);
+  padding-bottom: govuk-spacing(2);
+}
+
+.gem-c-cookie-banner--new .gem-c-cookie-banner__buttons {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.gem-c-cookie-banner--new .gem-c-cookie-banner__confirmation {
+  display: none;
+}
+
+.gem-c-cookie-banner__confirmation-message,
+.gem-c-cookie-banner__hide-button {
+  display: block;
+
+  @include govuk-media-query($from: tablet) {
+    display: inline-block;
+  }
+}
+
+.gem-c-cookie-banner__confirmation-message {
+  width: 90%;
```

Comment:
We have a mix or relative and absolute values defining the width of this element. Could we avoid this? Potentially by setting appropriate margins to this element or its parent?